lib/tts_plugin.dart

```diff
@@ -89,4 +113,25 @@ class TTSQueue {
     }
     return null;
   }
+
+  void manageTTSState() {
+    TtsModel? tts;
```

Comment:
`tts` is never actually set, so this will not disable TTS either. The button will indicate that TTS is enabled. Did you try this in the app?

Reply:
Yeah I couldn't get the notification, so I did do something wrong.